packages/kbn-user-profile-components/src/user_profiles_selectable.tsx

```diff
@@ -243,6 +248,7 @@ export const UserProfilesSelectable: FunctionComponent<UserProfilesSelectablePro
         onChange: onSearchChange,
         isLoading,
         isClearable: !isLoading,
+        name: USER_PROFILES_SELECTABLE_NAME,
```

Comment:
The design asks us to initially focus on the input search component. I _think_ this is how I'm supposed to do it. If there's a better way let me know. I'm exporting the string `USER_PROFILES_SELECTABLE_NAME` so I can use pass in the `initialFocus` prop.

Reply:
I would recommend using `searchProps.inputRef`. That ref can then be passed into the popover's `initialFocus` prop. That way there is no danger of running into naming clashes when multiple components are rendered on the page. 

This is something we should actually bake directly into the `UserProfilesPopover` component so you don't have to worry about this as a consumer. 

I've noticed you're not using that component. Is there a particular reason you're using `UserProfilesSelectable` directly instead of the popover? It gives you the correct paddings as well based on the designs. 

`UserProfilesSelectable` was supposed to be more of a low level building block for use cases that shouldn't be rendered inside a popover.

Reply:
For reference, PR that implements this functionality: https://github.com/elastic/kibana/pull/139161